doc.html

```diff
@@ -16,1281 +16,982 @@
+                    style="background-color: var(--pf-global--BackgroundColor--dark-100);">
+                    <nav class="pf-c-nav pf-m-start pf-m-end menu-wrapper" aria-label="Global">
+                        <button class="pf-c-nav__scroll-button left-paddle paddle hidden" aria-label="Scroll left">
+                            <i class="fas fa-angle-left" aria-hidden="true"></i>
+                        </button>
+                        <ul class="pf-c-nav__horizontal-list menu">
+                            <li class="pf-c-nav__item item">
+                                <a href="start.html" class="pf-c-nav__link">Get Started</a>
+                            </li>
+                            <li class="pf-c-nav__item item">
+                                <a href="doc.html" class="pf-c-nav__link pf-m-current">Documentation</a>
+                            </li>
+                            <li class="pf-c-nav__item item">
+                                <a href="learn.html" class="pf-c-nav__link">Learn More</a>
+                            </li class="pf-c-nav__item item">
+                            <a href="contact.html" class="pf-c-nav__link">Contact</a>
+                            </li>
+                        </ul>
+                        <button class="pf-c-nav__scroll-button right-paddle paddle hidden" aria-label="Scroll right">
+                            <i class="fas fa-angle-right" aria-hidden="true"></i>
+                        </button>
+                    </nav>
+                    <!-- // Will be added later  -->
+                    <!-- <div class="pf-c-input-group searchDiv">
                      <input class="pf-c-form-control searchInput" type="search" placeholder="search site" oninput="toggleSearchIcon()">
                      <button type="button" aria-label="search button for search input" class="searchBtn">
                        <i class="fas fa-search" aria-hidden="true"></i>
                      </button>
                      </div> -->
-               </div>
+                </div>
             </div>
         </header>
-        <main role="main" class="pf-c-page__main" tabindex="-1" style="background-color: white;">
-            <!-- <div class="content">
-                <div class="tab">
-                    <div class="pf-c-tabs" id="primary" style="align-items: center;margin-left: 10%;">
-                    <ul class="pf-c-tabs__list pf-l-flex" style="width: 100%;">
-                    <li class="tab-item pf-c-tabs__item pf-l-flex__item">
-                        <button class="tablinks active" onClick="openTab(event,'guide')">Agent User Guide</button>
-                    </li>
-                    <li class="tab-item pf-c-tabs__item pf-l-flex__item">
-                        <button class="tablinks" onClick="openTab(event, 'notes')">Release Notes</button>
-                    </li>
-                    </ul>
+        <div class="pf-c-page__sidebar">
```

Comment:
Looks like you basically dropped the `<main ...>` tag and the commented-out `"tab"` class div; but with the formatting the rest of this is very hard to compare. It looks like the `"pf-c-page__sidebar-body"` `div` is mostly the same, but I wish it were easier to compare side-by-side.

Reply:
@dbutenhof we had two separate tabs for installation doc and release notes. After we decided that the release notes will be tracked in GitHub instead, I deleted the release notes tab from here. Apart from that, yes, you are right. the `"pf-c-page__sidebar-body"` div is same. And the `<main...>` isn't dropped. it is moved down to wrap the `main section` (i.e the content of the installation guide)